File: Java/src/com/ouroboros/include/Terminal.java
package com.ouroboros.include;

public class Terminal {
    public static void gotoXY(int x, int y) {
        System.out.printf("\033[%d;%dH", y, x);
        System.out.flush();
    }

    public static void hideCursor() {
        System.out.print("\033[?25l");
        System.out.flush();
    }

    public static void showCursor() {
        System.out.print("\033[?25h");
        System.out.flush();
    }

    public static void clearDisplay() {
        System.out.print("\033[2J\033[H");
        System.out.flush();
    }
}

File: Kotlin/app/src/main/kotlin/org/example/terminal.kt
package org.example

object Terminal {
    fun gotoXY(x: Int, y: Int) {
        System.out.print("\u001B[" + y + ";" + x + "H")
    }

    fun hideCursor() {
        System.out.print("\u001B[?25l")
    }

    fun showCursor() {
        System.out.print("\u001B[?25h")
    }

    fun clearDisplay() {
        System.out.print("\u001B[2J\u001B[H")
    }
}